include/CoreGraphics/CGFont.h

```diff
@@ -19,6 +19,8 @@
 #import <CoreGraphics/CGGeometry.h>
 #import <CoreGraphics/CGDataProvider.h>
 
+typedef const struct __CGFont* CGFontRef;
+
```

Comment:
?

---
In reply to: [84347629](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r84347629) [](ancestors = 84347629)